src/app/rundown-overview/components/rundown-overview/rundown-overview.component.scss

```diff
@@ -1,3 +1,20 @@
 a {
   color: var(--blue-color);
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+td {
+  height: 55px;
+  border-bottom: solid 1px var(--gray-color);
```

Comment:
You are styling our table here. The styling of our table should happen in the TableComponent, so we can keep a consistent look throughout the application.

Reply:
Since the styling is due to us wanting a different feel of the overview page rather than the one we provide in the settings, this seemed the only way at the moment. In a future iteration we should redesign the front page, and not make use of this table component.